### ISSUE TITLE
Appearance > Node > Cant get partition based on simlar attributes

### ISSUE THREAD

**Issue by muhammadfaizan** [gephi/gephi#1386]
I have facebook group posts data, from netviz, I cant seem to color nodes based on their number of likes and similar attributes, the graph is of 50 posts only, so there is no such huge data to deal with. There is no partitioning Tab as well as no Ranking tab like in 0.8.2

**Comment by eduramiba** [comment]
Try to select an attribute in Appearance window, ranking and partition should then become available.

**Comment by muhammadfaizan** [comment]
yeah it worked after i ran modularity scan, and then selected attributes.
Gephi tutorials need to be updated
